app/config/const.go

```diff
@@ -43,4 +43,5 @@ const (
 	// UpgradeName gov proposal name
 	Upgrade2_2_0 = "2.2.0"
 	Upgrade2_3_0 = "2.3.0"
+	Upgrade2_4_0 = "2.4.0"
```

Comment:
Think @gregnuj wanted to rename this to `v2.4.0` to keep it aligned to our tagged releases.